### PR TITLE
Allow overrides for time API implementation 

### PR DESCRIPTION
**Pull request by rhythmize**
This PR makes l8w8jwt platform independent by allowing defining custom/ non-standard implementation of libc APIs like time. In case the user does not want to override, the dafult implementations fallback to libc ones, not breaking current functionality.

For context: I'm trying to use l8w8jwt to integrate token verification in ardupilot drone firmware (specifically for cube-orange hardware) which has non-standard implementation of time method.

Please feel free to leave some comments in case you have some. Much appreciated!

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/ARMmbed/mbedtls
 [submodule "lib/chillbuff"]
 	path = lib/chillbuff
-	url = https://github.com/GlitchedPolygons/chillbuff
+	url = https://github.com/rhythmize/chillbuff
```
Comment (rhythmize) [inline .gitmodules:6]: I had to make changes in chillbuff as well. I think it'll be better to first merge the chillbuff PR, if you don't have any concerns. And then I'll change the remote back to your repo for chillbuff

Comment (GlitchedPolygons) [reply .gitmodules:6]: I merged your chillbuff PR, you can revert the .gitmodules mod. and I'll look into this PR here now :)

**Review by GlitchedPolygons — CHANGES_REQUESTED** [review]
OK looks good, thank you for providing this PR :)

Will merge into master and produce a new release + version number bump after removal of the .gitmodules modification

File: .gitmodules (hunk 1 above)
Comment (rhythmize) [reply .gitmodules]: Reverted

**Comment by rhythmize** [comment]
Reverted the remote, please check it once again and merge if there are no concerns.

Thanks for the review.

**Review by GlitchedPolygons — APPROVED** [review]

**Pull request merged**